api/src/main/java/org/ccci/idm/user/Group.java

```diff
@@ -52,8 +53,10 @@ public int hashCode() {
         return Arrays.deepHashCode(new Object[]{this.path, this.name});
     }
 
+    private static final Joiner PATH_JOINER = Joiner.on(".").skipNulls();
+
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(this).add("path", this.path).add("name", this.name).toString();
+        return MoreObjects.toStringHelper(this).add("path", PATH_JOINER.join(path)).add("name", name).toString();
```

Comment:
before this change groups would stringify as `Group{path=[Ljava.lang.String;@5212494d, name=userAdmins}` now the path is actually included.